.travis.yml

```diff
@@ -0,0 +1,9 @@
+
+language: go
+go: 
+ - tip
```

Comment:
As a compromise, we can have both: `tip` and `1.6` (the version we're building the official releases -- right?).